packages/ng/forms/textarea-input/textarea-input.component.ts

```diff
@@ -14,11 +14,40 @@ import { NoopValueAccessorDirective } from '../noop-value-accessor.directive';
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TextareaInputComponent {
+	@ViewChild('clone')
+	clone?: ElementRef<HTMLElement>;
+
 	ngControl = injectNgControl();
 
+	#cdr = inject(ChangeDetectorRef);
+
 	@Input()
 	placeholder: string = '';
 
 	@Input()
 	rows?: number;
+
+	@Input({
+		transform: booleanAttribute,
+	})
+	autoResize = false;
+
+	@Input({
+		transform: booleanAttribute,
+	})
+	scrollIntoViewOnAutoResizing = false;
+
+	cloneValue = '';
+
+	updateScroll(value: string) {
+		this.cloneValue = value;
+		this.#cdr.detectChanges();
```

Comment:
Why this `detechChanges`? It should not be needed as `updateScroll` is called in an event binding.